app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationHelper.java

```diff
@@ -290,19 +289,14 @@ private static NotificationCompat.Builder newNotification(Context context, Notif
         PendingIntent eventResultPendingIntent = eventStackBuilder.getPendingIntent((int) account.getId(),
                 pendingIntentFlags(false));
 
-        Intent deleteIntent = new Intent(context, NotificationClearBroadcastReceiver.class);
-        deleteIntent.putExtra(ACCOUNT_ID, account.getId());
-        PendingIntent deletePendingIntent = PendingIntent.getBroadcast(context, summary ? (int) account.getId() : notificationId, deleteIntent,
-                pendingIntentFlags(false));
-
         NotificationCompat.Builder builder = new NotificationCompat.Builder(context, getChannelId(account, body))
                 .setSmallIcon(R.drawable.ic_notify)
                 .setContentIntent(summary ? summaryResultPendingIntent : eventResultPendingIntent)
-                .setDeleteIntent(deletePendingIntent)
                 .setColor(context.getColor(R.color.notification_color))
                 .setGroup(account.getAccountId())
                 .setAutoCancel(true)
                 .setShortcutId(Long.toString(account.getId()))
+                .setGroup(GROUP_KEY_GENERAL)
```

Comment:
setting group twice?

Reply:
Good point. I thik the account ID (String) is the better choice, so that notifications from different accounts don't get grouped together. I've made that change.